src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/storage/S3BulkResponseEntity.kt

```diff
@@ -2,10 +2,9 @@ package uk.gov.justice.digital.hmpps.managerecallsapi.storage
 
 import java.util.UUID
 
-data class S3BulkResponseEntity constructor(
+data class S3BulkResponseEntity(
```

Comment:
Was there before, but these should all be val not var

Reply:
Correct - I don't think we need this object tbh